pyproject.toml

```diff
@@ -57,6 +57,8 @@ dependencies = [
     "psutil >= 5.9.5", # may remove later
     "rich == 13.5.2",
     # TODO: Test on cpu-only machine if [cuda] dependency is an issue
+    # FIXME: Required for latest tritonclient 2.46.0
+    "grpcio>=1.64.0",
     "tritonclient[all] >= 2.45",
```

Comment:
Can you make a separate PR that will bump the following dependencies for the recent 24.05 release? This will go hand-in-hand as a follow-up to the `grpcio` change you're making in this PR:
```
    "genai-perf @ git+https://github.com/triton-inference-server/client.git@r24.04#subdirectory=src/c++/perf_analyzer/genai-perf",
    tritonclient[all] >= 2.45
```
to:
```
    "genai-perf @ git+https://github.com/triton-inference-server/client.git@r24.05#subdirectory=src/c++/perf_analyzer/genai-perf",
    tritonclient[all] >= 2.46
```

Reply:
Sorry to go back and forth on this -

In this PR, let's keep everything logically passing for 24.04:
- let's remove the grpcio version change
- let's lock the client version `tritonclient==2.45.0`

In a separate PR:
- add the grpcio version change 
- add the tritonclient and genai-perf changes I mentioned above

This way the pipeline should pass for this PR, and we can keep the next PR logically separate for 24.05 upgrade requirements